test/plugin/test_out_redis.rb

```diff
@@ -120,6 +121,22 @@ def test_write_with_insert_key_prefix
       assert_equal "3", d.instance.redis.hget("test.#{time}.1", "a")
     end
 
+    def test_write_with_ttl
+      d = create_driver CONFIG + %[
+        ttl 60
+      ]
+      time = event_time("2011-01-02 13:14:00 UTC")
```

Comment:
Ah, using this parameter with `allow_duplicate_key` is more actual usage, isn't it?